web/html/src/vendors/vendors.js

```diff
@@ -1,9 +1,10 @@
 const vendors = [
-    "babel-polyfill/dist/polyfill",
+    "@babel/polyfill/dist/polyfill",
     "react",
     "react-dom",
     "react-select",
     "prop-types",
+    "webpack/buildin/harmony-module.js"
```

Comment:
I wonder why webpack needs this file loaded to the browser :thinking: :thinking:  Are you sure we still need this? :thinking: I need to make some tests as well.

 I would say we don't want to include the file webpack/buildin/harmony-module.js on the production vendors bundle

Reply:
The reason I added it here was that it ended up in the `extravendors.notdeclared.bundle.js`. Not sure why that happened.

Reply:
I am curious why the well webpack needs that file now on the browser :thinking: I will try to check that out.

I was thinking about an approach like:

```javascript

const vendors = ({development}) => {

  const productionVendors = [
   "react",
   "react-dom",
  ]

  const developmentVendors = [
     "webpackThings"
   ]

  if ( development ) {
    return [...vendors, ...developmentVendors]
  }

  return productionVendors;
}
```